README.md

```diff
@@ -1,3 +1,190 @@
-# Codex Protocol | Codex Registry API
+# Codex Protocol | Codex Registry API _(service.codex-registry-api)_
```

Comment:
Is `service.codex-registry-api` redundant, since that is this repo's name?

Reply:
That was a requirement from the [standard-readme spec](https://github.com/RichardLitt/standard-readme/blob/master/spec.md#title), since the title (`Codex Registry API`) did not technically match the package / repo name (service.codex-registry-api):

> Title must match repository, folder and package manager names - or it may have another, relevant title with the repository, folder, and package manager title next to it in italics and in parentheses.

I agree it's kind of redundant though ¯\\\_(ツ)_/¯